kompose.spec

```diff
@@ -967,6 +973,18 @@ export LDFLAGS=%{ldflags}
 %doc RELEASE.md CHANGELOG.md README.md CONTRIBUTING.md ROADMAP.md code-of-conduct.md
 %{_bindir}/kompose
 
+%post
+if [ -z ${BASH+x} ];
+then
+  %{_bindir}/kompose completion zsh > %{_datadir}/zsh/site-functions/kompose;
+else
+  %{_bindir}/kompose completion bash > %{_datadir}/bash-completion/completions/kompose;
+fi
```

Comment:
so in `%build`, right after `%gobuild` then run `bin/kompose completion bash > file` and then in `%install` do something like `install` file into appropriate location. Under `%files` add them just like we do `%{_bindir}/kompose`